src/opengl/nodes/info.rs

```diff
@@ -0,0 +1,68 @@
+//! A `Node` that produces values based on information about the renderer and window
+
+use failure::Error;
+use std::collections::HashMap;
+use std::sync::mpsc::Receiver;
+use time::{self, Tm};
+
+use PointerEvent;
+use super::{Node, NodeInputs, NodeOutput};
+
+/// A `Node` that produces values based on information about the renderer and window
+pub struct InfoNode {
+    pointer_receiver: Receiver<PointerEvent>,
+    start: Tm,
+    resolution: [f32; 2],
+    pointer: [f32; 4],
+}
+
+impl InfoNode {
+    /// Create a new instance
+    pub fn new(pointer_receiver: Receiver<PointerEvent>, resolution: [f32; 2]) -> Self {
```

Comment:
Why do we send this node PointerEvents from within the renderer, and set up a list of pointer listeners there, instead of just sending generic `Event`s and maintaining a list of event listeners, such that other nodes could use the message bus in other ways in the future?

Reply:
hmm interesting point, would that be the same thing except with generic `RendererEvent`s?

Reply:
yes, that's what i meant; sorry :stuck_out_tongue:

Reply:
Gotcha. I'll do that real quick then.